diffkemp/simpll/Utils.cpp

```diff
@@ -49,6 +49,18 @@ const Function *getCalledFunction(const Value *CalledValue) {
     return fun;
 }
 
+Function *getCalledFunction(Value *CalledValue) {
```

Comment:
This can be implemented by (no code duplication):

    return const_cast<Function *>(getCalledFunction(const_cast<const Value *>(CalledValue)))